buildmaster/backend/assets/bin/missing_ports.sh

```diff
@@ -10,7 +10,7 @@ fi
 
 set -e
 
-HAIKUPORTER=${HAIKUPORTER:-haikuporter}
```

Comment:
I'm not sure how this variable in `sh` script works. I need to check if it is ok now.

Reply:
According to https://unix.stackexchange.com/questions/122845/using-a-b-for-variable-assignment-in-scripts it looks like this change is ok, but please check it.